wave_lang/kernel/ops/wave_ops.py

```diff
@@ -130,9 +130,13 @@ def write(
 
 def debug_log_write(
     register_: "Register",
-    elements_per_thread: Optional[IndexExpr | int] = None,
-    mapping: Optional[IndexMapping] = None,
-    mapping_dynamic_vals: "Register" | tuple["Register", ...] = (),
```

Comment:
Why are these not needed anymore?

Reply:
They were actually removed from the actual operation in the previous commit before it landed, but I missed updating the signature.